discovery_engine_core/providers/src/client.rs

```diff
@@ -212,6 +227,15 @@ impl Client {
     }
 }
 
+/// Default `from` value for newscatcher news queries
+pub fn default_from() -> String {
+    let from = Utc::today() - chrono::Duration::days(3);
+    from.format("%Y/%m/%d").to_string()
+}
+
+/// Default `when` value for newscatcher headline queries
+pub const DEFAULT_WHEN: Option<&'static str> = Some("3d");
```

Comment:
🎨 we could make the type consistence with the return type of `default_from` by removing `Option`

```suggestion
pub const DEFAULT_WHEN: &str = "3d";
```